packages/web-components/src/components/cbp-notice/cbp-notice.scss

```diff
@@ -0,0 +1,60 @@
+/**
+ * @prop --cbp-list-padding: var(--cbp-space-4x);
+*/
+
+:root {  
+  --cbp-notice-background: var(--cbp-color-info-lighter);
+  --cbp-notice-background-dark: var(--cbp-color-info-dark);
+  --cbp-notice-border-left-color: var(--cbp-color-info-base);
+  --cbp-notice-border-left-color-dark: var(--cbp-color-info-light);
+}
+
+
+[data-cbp-theme=light] cbp-notice[context*=dark]:not([context=light-always]),
+[data-cbp-theme=dark] cbp-notice:not([context=dark-inverts]):not([context=light-always]) {
```

Comment:
You'll need variables for the text (heading and content if they are not the same), because the text isn't changing when you change the context alone (not the OS dark/light mode).

Reply:
updated